Agda.cabal

```diff
@@ -550,6 +553,9 @@ library
 
   other-modules:    Paths_Agda
 
+  ghc-options: -fexpose-all-unfoldings
+               -fspecialise-aggressively
```

Comment:
Should these options also be added to the `test-suite` target below? All the other `ghc-options` are the same, except `-threaded`, which I was considering making a PR for.

It'll be nice once Cabal 2.2 can be used so these could be consolidated with [the newer `common`/`import` stanzas](https://www.haskell.org/cabal/users-guide/file-format-changelog.html#cabal-version-2-2)…

Reply:
Done